server/package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name": "odyssey-server",
-  "version": "1.0.0",
-  "private": true,
-  "scripts": {
```

Comment:
Lets keep tab widths to 2 spaces

Reply:
do we have a prettier config @WilhelmWillie ?

Reply:
yeah we do, not sure why it didn't get caught here (see: https://github.com/HackSC/odyssey/blob/staging/package.json)